src/main/java/run/halo/app/model/params/PostQuery.java

```diff
@@ -20,11 +22,22 @@
     /**
      * Post status.
      */
-    private PostStatus status;
+    private Set<PostStatus> status;
```

Comment:
Because this field type is a `Set`, I suggest naming field name to `statuses`.

Reply:
but status is an uncountable noun